src/modules/cmdpal/ext/Microsoft.CmdPal.Ext.Apps/Programs/Win32Program.cs

```diff
@@ -201,6 +201,16 @@ public List<IContextItem> GetCommands()
             });
         }
 
+        if (AppType == ApplicationType.ShortcutApplication || AppType == ApplicationType.ApprefApplication || AppType == ApplicationType.Win32Application)
```

Comment:
This should probably be the last in the list of commands. (excluding pin/unpin & show details)